admin/views/availability/admin-window-unavailable.ejs

```diff
@@ -0,0 +1,52 @@
+<div class="grid-row">
+    <div class="column-full">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row" aria-describedby="lead-paragraph">
+    <div class="column-two-thirds">
+
+        <%- partial('partials/_session_expiry') %>
+
+        <header class="page-header">
+            <h1 class="heading-xlarge">
+                <% if(checkWindow && currentDate.isBefore(checkWindow.adminStartDate)) { %>
+                The service is unavailable
```

Comment:
I'm not sure this check is giving us much...

Reply:
In addition whichever behavior is picked up for this shouldn't it be part of a presenter?

Reply:
The prototypes specify two difference messages hence this code.